website/docs/design/type-parameters.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Type parameters
+---
+
+You can customize the behavior with optional type parameters.
+
+Type constraint | Default | Possible values
+:--- | :--- | :---
+`HTMLElement` | `HTMLButtonElement` | Any subclass of `HTMLElement`
+
+```js
+const { buttonProps, itemProps, isOpen } = useDropdownMenu<HTMLDivElement>(3);
+```
+
+```jsx
+<button {...buttonProps} id='menu-button'>
```

Comment:
You used a `HTMLDivElement` as the button type above but in the example you're using a `button` element.